chricar_product_gtin/tests/__init__.py

```diff
@@ -0,0 +1 @@
+from . import test_product_gtin
```

Comment:
I guess it's the same as here:
https://github.com/savoirfairelinux/openerp-adhoc-product/pull/4